edge-apps/countdown-timer/README.md

```diff
@@ -0,0 +1,61 @@
+# Screenly Countdown Timer App
+
+![Countdown Timer Screen](./static/images/countdown_timer_screen.jpg)
+
+## tl;dr
+
+```bash
+$ cd edge-apps/countdown-timer
+$ screenly edge-app create \
+    --name countdown-timer \
+    --in-place
+$ screenly edge-app upload
+[...] # You can tweak settings here.
+$ screenly edge-app version promote --revision=X
+
+# Alternatively, you can use --latest in place of --revision.
+```
+
+## Tweaking the settings
+
+### `timer`
+
+To set the countdown timer, we can use the `timer` settings.
+
+```bash
+$ Screenly edge-app setting set timer=2024-03-14T00:00:00
+# A relatively long console output...
+Edge app setting successfully set.
+```
+where `2024-03-14T00:00:00` your countdown timer limit with `YYYY-MM-DDTHH:MM:SS` format.
```

Comment:
It's important to point out that this is the countdown to a local timezone where the screen is. You can use `override_timezone` to pin a specific time zone across screens that span multiple time zones.

Reply:
Yeah, Currently it's reading the local timezone and the countdown is based on that.

Reply:
Tested and works fine.